src/app/productConfig/productConfigModal/productConfig.modal.tpl.html

```diff
@@ -38,24 +38,53 @@ <h4 class="panel-title border-bottom-small" translate>
             Gift Amount
           </h4>
           <div class="panel-body pt0">
-            <div ng-if="$ctrl.suggestedAmounts">
-              <div class="radio radio-custom-amount" ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
+            <div ng-if="$ctrl.useSuggestedAmounts">
+              <div class="radio radio-custom-amount"
+                   ng-repeat="suggested in $ctrl.suggestedAmounts | orderBy:'order'">
                 <label>
-                  <input name="amount"
-                         ng-model="$ctrl.customAmount"
-                         ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
+                  <input name="suggestedAmount"
                          type="radio"
-                         ng-value="suggested.amount"
-                         required>{{suggested.amount | currency:'$':2}} {{suggested.label}}
+                         ng-click="$ctrl.changeAmount(suggested.amount)"
+                         ng-checked="!$ctrl.customInputActive && $ctrl.itemConfig.amount === suggested.amount">
+                  {{suggested.amount | currency:'$':2}} {{suggested.label}}
                 </label>
               </div>
+              <div class="radio radio-custom-amount form-inline" ng-class="{'has-error': ($ctrl.itemConfigForm.amount | showErrors)}">
+                <div class="form-group">
+                  <label>
+                    <input name="suggestedAmount"
+                           type="radio"
+                           ng-checked="$ctrl.customInputActive">
+                    <input class="form-control form-control-subtle"
+                           name="amount"
+                           type="text"
+                           step="1"
+                           tabindex="-1"
+                           ng-model="$ctrl.customAmount"
+                           ng-change="$ctrl.changeCustomAmount($ctrl.customAmount)"
+                           ng-focus="$ctrl.customInputActive = true;"
+                           ng-required="$ctrl.customInputActive"
+                           placeholder="{{'Other' | translate}}">
+                  </label>
+                  <div role="alert" ng-messages="$ctrl.itemConfigForm.amount.$error"
+                       ng-if="($ctrl.itemConfigForm.amount | showErrors)">
+                    <div class="help-block" ng-message="pattern" translate>Your gift must be a valid dollar amount</div>
+                    <div class="help-block" ng-message="required" translate>Amount must be not empty</div>
+                    <div class="help-block" ng-message="minimum" translate>Amount must be at least {{ 1 | currency }}</div>
```

Comment:
`All localizable AngularJS components depend on locale-specific rule sets managed by the $locale service.` Seems like you have to call that explicitly and have loaded other locales. Also see https://docs.angularjs.org/guide/i18n#currency-symbol. Is that what you get from there? I could be wrong. The docs aren't the clearest.